docs/framework/data/adonet/ef/language-reference/divide-entity-sql.md

```diff
@@ -8,7 +8,7 @@ Divides one number by another.
   
 ## Syntax  
   
-```  
+```sql  
 dividend / divisor  
 ```  
   
```

Comment:
Some of the articles on this PR as well have the C# example in the bottom that should be SQL. Can you review all of them to make sure?